### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by yk2274** [kubernetes-client/python#2143]
This line is giving ssl_ca_cert the file path instead of the content of the file

https://github.com/kubernetes-client/python/blame/36cfbe68a509d9b9d33395b22b6fa94d7d46c30f/kubernetes/base/config/incluster_config.py#L85C4-L85C4

**Comment by ggree1** [comment]
Yeah.. I there a way to pass cert data instead of ca file path for config.ssl_ca_cert?

**Comment by roycaihw** [comment]
Agreed it seems to be a bug that the code is only passing the filename instead of reading the content. Would be good if we can have a test to catch the bug. 

/help

**Comment by ddfreiling** [comment]
This is still a bug, cannot use ansible k8s_info on a k3s node where the ~/.kube/config just contains `client-certificate-data` and `certificate-authority-data`

**Comment by abhi-dev91** [comment]
I also need the solution on the same, we have the requirement to pick the CA certificates from the databases as we are managing thousands of kubernetes clusters and could not maintation CA or kubeconfig file individually for each.

**Comment by darshanip** [comment]
@roycaihw I can help on this one.
